examples/matching-engine/tests/transaction.rs

```diff
@@ -76,14 +77,18 @@ async fn single_transaction() {
     let mut matching_chain = validator.new_chain().await;
     let admin_account = AccountOwner::from(matching_chain.public_key());
 
-    let fungible_bytecode_id_a = user_chain_a.publish_bytecodes_in("../fungible").await;
-    let fungible_bytecode_id_b = user_chain_b.publish_bytecodes_in("../fungible").await;
+    let fungible_bytecode_id_a = user_chain_a
+        .publish_bytecodes_in::<FungibleTokenAbi, fungible::Parameters, InitialState>("../fungible")
```

Comment:
It's a bit inconsistent that `Parameters` is qualified with `fungible` but the other two are imported.